crates/data_components/src/graphql/client.rs

```diff
@@ -428,14 +451,27 @@ impl PaginationParameters {
         let regex =
             Regex::new(&pattern).unwrap_or_else(|_| panic!("Invalid regex query resource pattern"));
 
+        let other_arguments = self
+            .other_arguments
+            .iter()
+            .map(|arg| format!("{}: {}", arg.name, arg.value))
```

Comment:
nit:
 - Move `format!("{}: {}", arg.name, arg.value` to `FieldArgument`.
 - Move `.collect::<Vec<String>>().join(", ")` to ` PaginationParameters`.